k8s/helm-chart/dynatrace-gcp-function/templates/function-config.yaml

```diff
@@ -40,6 +40,7 @@ data:
   DYNATRACE_LOG_INGEST_ATTRIBUTE_VALUE_MAX_LENGTH: {{ .Values.logIngestAttributeValueMaxLength | quote }}
   DYNATRACE_LOG_INGEST_REQUEST_MAX_EVENTS: {{ .Values.logIngestRequestMaxEvents | quote }}
   DYNATRACE_LOG_INGEST_REQUEST_MAX_SIZE: {{ .Values.logIngestRequestMaxSize | quote }}
+  DYNATRACE_LOG_INGEST_SENDING_WORKER_EXECUTION_PERIOD: {{ .Values.logIngestSendingWorkerExecutionPeriod | quote }}
```

Comment:
I've checked other flags and found deference with src/lib/fast_check.py:36 
LOGS_SUBSCRIPTION_PROJECT is missing

Reply:
this variable is taken in log_forwarder_variables.py as GCP_PROJECT so it's not mandatory
```python
LOGS_SUBSCRIPTION_PROJECT = os.environ.get("GCP_PROJECT", os.environ.get("LOGS_SUBSCRIPTION_PROJECT", None))
```

Reply:
We shouldn't add `LOGS_SUBSCRIPTION_PROJECT` to deployment script, it was used in the first version of gcp log forwarder deployment script, here is used as a fallback.